bindings/lua/src/lib.rs

```diff
@@ -65,6 +65,8 @@ fn operator_new<'a>(
 
     let operator = lua.create_table()?;
     operator.set("_operator", op)?;
+    operator.set("limit", lua.create_function(operator_limit)?)?;
+    operator.set("with_limit", lua.create_function(operator_with_limit)?)?;
```

Comment:
Those API is not ready to be exposed.

Reply:
Sorry, Can I get the exposed API list doc?

Reply:
Sorry, but there is no such thing. 

By not exposing this API, my idea is that we don't need to try and expose all APIs on the Lua side. Instead, we should focus on the most commonly used APIs such as read/write/list/delete, etc. Our first release should be built with this in mind - finding our first user and adding more APIs based on their needs.

Reply:
> Our first release should be built with this in mind - finding our first user and adding more APIs based on their needs.

Do you know of any Lua open source projects that might be interested in using OpenDAL?

Reply:
> > Our first release should be built with this in mind - finding our first user and adding more APIs based on their needs.
> 
> Do you know of any Lua open source projects that might be interested in using OpenDAL?

I will try to find potential projects to use opendal lua binding after I finished some doc work.